packages/admin-api-client/src/admin-api-client.ts

```diff
@@ -88,17 +90,17 @@ export function createAdminApiClient({
   const getHeaders = generateGetHeaders(config);
   const getApiUrl = generateGetApiUrl(config, apiUrlFormatter);
 
-  const getGQLClientParams = generateGetGQLClientParams({
+  const getGQLClientParams = generateGetGQLClientParams<AdminOperations>({
     getHeaders,
     getApiUrl,
   });
 
-  const fetch = (...props: ApiClientRequestParams) => {
+  const fetch: ApiClientFetch<AdminOperations> = (...props) => {
```

Comment:
```suggestion
  const fetch: AdminApiClient['fetch'] = (...props) => {
```

This might make more sense to devs since the `ApiClientFetch` is a generic type for all API clients in general.